### ISSUE TITLE
accessibility audit

### ISSUE THREAD

**Issue by ebenenglish** [boston-library/commonwealth-vlr-engine#112]
This is a placeholder ticket. more details TK. 

We need to improve the accessibility of `digitalcommonwealth.org`.

WAVE tool could be useful for identifying errors and areas for improvement: [​https://wave.webaim.org/](https://wave.webaim.org/).

See also many recent Code4Lib presentations on this issue, should be some useful information there as well.
